video/out/w32_common.c

```diff
@@ -2275,6 +2295,12 @@ static int gui_thread_control(struct vo_w32_state *w32, int request, void *arg)
     case VOCTRL_BEGIN_DRAGGING:
         w32->start_dragging = true;
         return VO_TRUE;
+    case VOCTRL_SHOW_MENU:
+        PostMessageW(w32->window, WM_SHOWMENU, 0, 0);
+        return VO_TRUE;
+    case VOCTRL_UPDATE_MENU:
+        mp_win32_update_menu(w32->menu_ctx, w32->menu, (struct mpv_node *)arg);
```

Comment:
nit, but I would name it `mp_win32_menu_update` with the `<prefix>_<action>` style, all of them are "menu" anyway.

Similarly for 
`mp_win32_menu_init`
`mp_win32_menu_show`